src/main/java/core/basesyntax/MainApp.java

```diff
@@ -8,9 +8,24 @@
  * машина начала свою работу.
  * - Переопределить метод stopWork() и вывести в консоль строку с сообщением что та или иная
  * машина прекратила свою работу.
- * - В классе Main создать добавить по одному екземпляру каждого класса.
- * - Отправить все машини на работу и остановить работу всех машин.
+ * - В классе Main создать добавить по одному экземпляру каждого класса.
+ * - Отправить все машины на работу и остановить работу всех машин.
  */
 public class MainApp {
+    public static void main(String[] args) {
 
+        Bulldozer bulldozer = new Bulldozer();
```

Comment:
Be more careful, according to check list and slack message reference to object should be abstract 
Example: 
Animal cat = new Cat();
in our case it should be 
Machine bulldozer = new Bulldozer() 
etc